From 24cf9fcef1309ec28fcc09cd6d71e0b5d0370bed Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Wed, 21 Aug 2013 11:51:31 +0800
Subject: [PATCH 408/435] ENGR00273601 mxc v4l2 tvin: remove an annoying
 invalid log

This patch removes an annoying invalid error log which comes
even if the test runs successfully.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_tvin.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_tvin.c b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
index a54a40b..114cb1e 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_tvin.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
@@ -303,8 +303,6 @@ int v4l_capture_setup(void)
 	g_in_width = fmt.fmt.pix.width;
 	g_in_height = fmt.fmt.pix.height;
 
-	printf("VIDIOC_G_FMT failed\n");
-
 	memset(&req, 0, sizeof (req));
 
 	req.count               = g_capture_num_buffers;
-- 
1.8.0

